Sources/Configurations/CreditCardConfiguration.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  CreditCardConfiguration.swift
+//  FormValidatorSwift
+//
+//  Created by Aaron McTavish on 04/01/2017.
+//  Copyright © 2017 ustwo. All rights reserved.
+//
+
+import Foundation
+
+
+/// Credit card types that are supported by `CreditCardCondition`. There is a `regex` property that returns the regex for validating that credit card number.
+public struct CreditCardType: OptionSet, CustomStringConvertible {
+    public var rawValue: Int
+    
+    public init(rawValue: Int) {
+        self.rawValue = rawValue
+    }
+    
+    static let americanExpress    = CreditCardType(rawValue: 1 << 0)
+    static let dinersClub         = CreditCardType(rawValue: 1 << 1)
+    static let discover           = CreditCardType(rawValue: 1 << 2)
+    static let jcb                = CreditCardType(rawValue: 1 << 3)
+    static let maestro            = CreditCardType(rawValue: 1 << 4)
+    static let mastercard         = CreditCardType(rawValue: 1 << 5)
+    static let visa               = CreditCardType(rawValue: 1 << 6)
+    
+    static let all: CreditCardType = [.americanExpress, .dinersClub, .discover, .jcb, .maestro, .mastercard, .visa]
+    static let allArray: [CreditCardType] = [.americanExpress, .dinersClub, .discover, .jcb, .maestro, .mastercard, .visa]
+    
+    /**
+     Literal description for set of credit cards, separated with commas and single space.
+     */
+    public var description: String {
+        
+        var result = ""
+        
+        add(description: "American Express", forCardType: CreditCardType.americanExpress, toResultString: &result)
```

Comment:
May or may not be worth it. Many languages do not localise company names. Also, this was just moving the existing OptionSet from the Condition to the new Configuration.